_schemaV2.graphql

```diff
@@ -6944,18 +6944,16 @@ type LocationEdge {
 
 scalar Long
 
-# A lot in an auction containing merged Sale artwork and Lot state data.
+# A lot in an auction containing merged SaleArtwork and LotState data, stitched from causality.
 type Lot implements Node {
   # A globally unique ID.
   id: ID!
 
   # A type-specific ID likely used as a database ID.
   internalID: ID!
+  lot: AuctionsLotState
```

Comment:
correct me if i'm wrong, but this _is_ a breaking change, isn't it? Eigen clients will need to adapt to check lot state is present before rendering anything that requires it.

Reply:
Yup, this is the follow-up I mentioned. The MyBids component will need to check if null in the loop, but its not a breaking _schema_ change since the same query fields are still in place.

Reply:
I think maybe we are talking about two slightly different different things. Isn't it possible this change would break eigen right here if a current app version loaded a lot with no `lot` state? https://github.com/artsy/eigen/blob/fcdad03ed67580286af2af27cf69a5eb71243c6e/src/lib/Scenes/MyBids/Components/WatchedLot.tsx#L18-L20

Reply:
Ahh, I see what you mean. Here's a q. Outside of weird staging env, is it possible for there to be a lot with no lot state?

Reply:
Almost certainly not!